code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -1040,6 +1040,21 @@
 	filling_color = "#EDDD00"
 	list_reagents = list("nutriment" = 1, "vitamin" = 1)
 
+/obj/item/reagent_containers/food/snacks/macaroni
+	name = "Macaroni twists"
+	desc = "These are little twists of raw macaroni."
+	icon_state = "macaroni"
+	filling_color = "#EDDD00"
+	list_reagents = list("nutriment" = 1, "vitamin" = 1)
+
+/obj/item/reagent_containers/food/snacks/macncheese
+	name = "Macaroni cheese"
+	desc = "One of the most comforting foods in the world. Apparently."
+	trash = /obj/item/trash/snack_bowl
+	icon_state = "macncheese"
+	filling_color = "#ffe45d"
+	list_reagents = list("nutriment" = 5, "vitamin" = 2, "cheese" = 4)
+
```

Comment:
Note to self: add flavour message here

Reply:
Taste messages are on reagents, not food items. You taste the reagents in the item, but not the item itself.

Reply:
Aaaaah, right.

Reply:
That said... it might not be a terrible idea to allow food items to "override" reagent tastes with one of their own. Corn chips and popcorn don't taste that similar, despite both being corn, after all.